cmd/experimental/kjobctl/pkg/builder/slurm_builder_test.go

```diff
@@ -84,6 +86,17 @@ func TestUnmaskFilenameFunction(t *testing.T) {
 				return
 			}
 
+			tmpl, err := template.ParseFS(slurmTemplates, "templates/*")
+			if err != nil {
+				t.Fatal(err)
+			}
+
+			var unmaskFilenameFunction bytes.Buffer
+
+			if err := tmpl.ExecuteTemplate(&unmaskFilenameFunction, "slurm_unmask_filename.sh.tmpl", nil); err != nil {
+				t.Fatal(err)
+			}
+
```

Comment:
Why not embed the file in a string or []byte if no tempting is done?

Reply:
Or just put the content in the entry point script template.

Reply:
Not sure I got question correct. You mean to create separate embed variable for this case?

Reply:
There is no tempting to be done in the content of `cmd/experimental/kjobctl/pkg/builder/templates/slurm_unmask_filename.sh.tmpl` no `{{ .... }}`, maybe you can just copy its content in the entry point file.

Reply:
Done